opm/simulators/wells/GasLiftStage2_impl.hpp

```diff
@@ -1140,4 +1142,4 @@ updateRates(const std::string &well_name)
     this->alq += delta_alq;
 }
 
-
+}
```

Comment:
Would you mind tagging this closing brace with `// namespace Opm`?

Reply:
done.